java/ingest/src/main/java/sleeper/ingest/job/IngestJobRunner.java

```diff
@@ -116,26 +116,24 @@ public IngestResult ingest(IngestJob job) throws InterruptedException, IteratorE
                         + ". This file will be ignored and will not be ingested.", pathString);
             }
         }
+        // Run the ingest
+        try (CloseableIterator<Record> recordIterator = new ConcatenatingIterator(inputIterators);
+             BufferAllocator bufferAllocator = new RootAllocator()) {
 
-        // Concatenate iterators into one iterator
-        CloseableIterator<Record> concatenatingIterator = new ConcatenatingIterator(inputIterators);
-
-        // Get StateStore for this table
-        StateStore stateStore = stateStoreProvider.getStateStore(tableProperties);
+            IngestCoordinatorFactory factory = IngestCoordinatorFactory.builder()
+                    .objectFactory(objectFactory)
+                    .stateStoreProvider(stateStoreProvider)
+                    .localDir(localDir)
+                    .s3AsyncClient(s3AsyncClient)
+                    .hadoopConfiguration(hadoopConfiguration)
+                    .bufferAllocator(bufferAllocator)
+                    .instanceProperties(instanceProperties)
+                    .tablePropertiesProvider(new FixedTablePropertiesProvider(tableProperties))
```

Comment:
FixedTablePropertiesProvider is in the test code. It looks like the only reason this compiles is because the test jar is set as a production dependency. It's not needed.